### PR TITLE
Established django project skeleton. Done as a group.

### LINKED ISSUE #2: As a developer, I want to learn Django and use it as the backend of the application.

### LINKED ISSUE #3: As a developer, I want to be able to review Python and use it to set up the front end of the website.

### PR DESCRIPTION
**Pull request by sharletclaros**
Includes skeleton HTML files and basic path definitions.

### CODE REVIEW

File: sadcowpng/sharedspaces/urls.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+urlpatterns = [
+    path('', views.index, name='index'),
+    # account page renders based on user input role
+    path('account/', views.account, name='account'),
+    path('login/', views.login, name='login'),
+    path('signup/', views.signup, name='signup'),
+    path('create/', views.createspace, name='createspace'),
+
+]
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/urls.py]: nit: eof character missing

**Review by bishalr1 — DISMISSED** [review]
Looks good.

File: sadcowpng/sharedspaces/templates/sharedspaces/signup.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Signup</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/templates/sharedspaces/signup.html]: nit: eof char missing

File: sadcowpng/sharedspaces/templates/sharedspaces/login.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Login</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/templates/sharedspaces/login.html]: nit: eof char missing

File: sadcowpng/sadcowpng/urls.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
+from django.contrib import admin
+from django.urls import include, path
+#path names are debatable
+urlpatterns = [
+    path('', include('sharedspaces.urls')),
+    path('admin/', admin.site.urls),
+]
```
Comment (jimmy-coleman) [inline sadcowpng/sadcowpng/urls.py]: nit: eof char missing

File: sadcowpng/sharedspaces/templates/sharedspaces/account.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Account</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/templates/sharedspaces/account.html]: nit: eof char missing

File: sadcowpng/sharedspaces/templates/sharedspaces/index.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Shared Spaces</title>
+</head>
+<body>
+    <p> This is the main page </p>
+</body>
+</html>
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/templates/sharedspaces/index.html]: nit: eof char missing

**Review by bishalr1 — DISMISSED** [review]
LGTM. Looks good so far. I made sure no private keys were leaked.

File: sadcowpng/sadcowpng/urls.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
+from django.contrib import admin
+from django.urls import include, path
+#path names are debatable
```
Comment (dmercanti) [inline sadcowpng/sadcowpng/urls.py]: Put a space between # and path in the comment here (line 17)

File: sadcowpng/sharedspaces/urls.py (hunk 1 above)
Comment (dmercanti) [inline sadcowpng/sharedspaces/urls.py]: put a newline at the end of the file

File: sadcowpng/sadcowpng/urls.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
+from django.contrib import admin
+from django.urls import include, path
```
Comment (jimmy-coleman) [inline sadcowpng/sadcowpng/urls.py:18]: nit: PEP8 specifies 2 blank lines after imports

File: sadcowpng/sadcowpng/urls.py (hunk 7 above)
Comment (jimmy-coleman) [inline sadcowpng/sadcowpng/urls.py]: Consider removing this comment or updating it to be more meaningful

**Review by dmercanti — DISMISSED** [review]
Comments on sadcowpng/sadcowpng/urls.py and sadcowpng/sharedspaces/urls.py, both of them are stupid stylized changes. Everything else lgtm

File: sadcowpng/sharedspaces/urls.py [hunk 9]
```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index'),
+    # account page renders based on user input role
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/urls.py]: two cents: i believe this comment would be more appropriately placed in the `account` function in `sharedspaces/views.py`

File: sadcowpng/sharedspaces/views.py [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+def account(request):
+    return HttpResponse("Here is your account page!")
+
+def login(request):
+    return HttpResponse("Login here")
+
+def signup(request):
+    return HttpResponse("Sign up here")
+
+def createspace(request):
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/views.py]: nit: PEP8 specifies underscores between words for function names

File: sadcowpng/sharedspaces/views.py [hunk 11]
```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+
+from django.http import HttpResponse
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/views.py]: nit: not specified by PEP8, but generally accepted best-practices have imports places together sorted alphabetically. pep8 specifies two lines below import statements

File: sadcowpng/sharedspaces/views.py [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+#Shared Spaces Home Page
+def index(request):
```
Comment (jimmy-coleman) [inline sadcowpng/sharedspaces/views.py:6]: nit: PEP8 specifies two blank lines between functions

**Review by jimmy-coleman — CHANGES_REQUESTED** [review]
Stylistic consistency change requests mostly

**Review by btruong9 — DISMISSED** [review]
Other than stylistic changes mentioned by other reviewers, looks good.

File: sadcowpng/sharedspaces/views.py (hunk 11 above)
Comment (sharletclaros) [reply sadcowpng/sharedspaces/views.py]: Not updated. Just following PEP8 standards.

**Review by jimmy-coleman — APPROVED** [review]
lgtm

**Pull request merged**